new-integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/impl/Pod.java

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) 2020, Oracle Corporation and/or its affiliates.
+// Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+package oracle.weblogic.kubernetes.actions.impl;
+
+import io.kubernetes.client.openapi.models.V1Pod;
+import io.kubernetes.client.openapi.models.V1PodList;
+import oracle.weblogic.kubernetes.actions.impl.primitive.Kubernetes;
+
+public class Pod {
+
+  /**
+   * Delete Kubernetes Pod.
+   *
+   * @param name name of the pod
+   * @param namespace name of namespace
+   * @return true if successful, false otherwise
+   */
+  public static boolean deletePod(String name, String namespace) {
+    return Kubernetes.deletePod(name, namespace);
+  }
+
+  /**
+   * List Kubernetes pods in a namespace.
+   *
+   * @param namespace name of namespace
+   * @param labelSelectors with which pods are decorated
+   * @return V1PodList list of pods
+   */
+  public static V1PodList listPods(String namespace, String labelSelectors) {
+    return Kubernetes.listPods(namespace, labelSelectors);
+  }
+
+  /**
+   * Get a pod's log.
+   *
+   * @param podName name of the pod
+   * @param namespace name of the namespace
+   * @return log as a String
+   */
+  public static String getPodLog(String podName, String namespace) {
```

Comment:
this assumes a pod only has one container
there are likely going to be cases where we want to get the log from a specific container.
consider introducing an optional container parameter.
if the pod only has one container, default to that one.

Reply:
Good suggestion. In WKO all the WebLogic pod has a single container. This method can be used to verify id the pod log has any exception. We can open a JIRA to enhance this utility. In this PR I just made the method publically available through TestAction.